README.md

```diff
@@ -145,26 +144,32 @@ Our exception tracking tool of choice is [Rollbar](https://rollbar.com). The
 [rollbax](https://github.com/elixir-addicts/rollbax) hex package is included and
 configured to capture errors **only in the production environment**. To complete
 the Rollbar configuration, you will need to create a project in rollbar, and add
-the access token and application environment to the production environment
-variables. On heroku, you can do this with the following command:
+the access token to the production environment variables. On heroku, you can do
+this with the following command:
 
 ```bash
-$ heroku config:set APPLICATION_ENVIRONMENT=production ROLLBAR_ACCESS_TOKEN=#{access token copied from rollbar}
+$ heroku config:set ROLLBAR_ACCESS_TOKEN=#{access token copied from rollbar} DEPLOY_TAG=#{optional deploy tag to identify specific production environment}
 ```
 
-Once the environment variables are in place and the rollbar project is setup,
+Once the environment variables are in place and the rollbar project is set up,
 rollbar notifications can be created with:
```

Comment:
Since the presence of a `DEPLOY_TAG` will be the exception and not the rule, I think it’s clear enough as is. I don’t feel too strong about that though, so I could be swayed.